test/keycloak/irods-http-api-keycloak.Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM quay.io/keycloak/keycloak:latest as builder
```

Comment:
Add comments above the `FROM` lines which explain what the whole section is for. The comments will also serve as separators for the multi-stage build.